### PR TITLE
[ASA-364][RV] add personal details form

### PR DESCRIPTION
**Pull request by rvijayan**

### CODE REVIEW

File: app/uk/gov/hmrc/agentoverseasapplicationfrontend/views/personal_details.scala.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            <legend>
+                <h1 class="heading-large">@Messages("personalDetails.title")</h1>
+                <p>@Messages("personalDetails.p1")</p>
+                <p class="faded-text">@Messages("personalDetails.p2")</p>
+            </legend>
+            <div class="multiple-choice" data-target="ninoRadioInput">
+                <input id="variant_nino" type="radio" name="personalDetailsChoice" @if(personalDetailsForm.value.exists(_.choice.value == "nino")){checked} value="nino" >
+                <label for="variant_nino">@Messages("personalDetails.form.nino")</label>
+            </div>
+            <div class="panel panel-border-narrow js-hidden" id="ninoRadioInput">
```
Comment (arkowalski) [inline app/uk/gov/hmrc/agentoverseasapplicationfrontend/views/personal_details.scala.html]: this div and above have same id

File: it/uk/gov/hmrc/agentoverseasapplicationfrontend/controllers/ApplicationControllerISpec.scala [hunk 2]
```diff
@@ -423,6 +427,74 @@ class ApplicationControllerISpec extends BaseISpec {
     }
   }
 
+  "GET /personal-details" should {
```
Comment (arkowalski) [inline it/uk/gov/hmrc/agentoverseasapplicationfrontend/controllers/ApplicationControllerISpec.scala]: test for form prepopulation

File: it/uk/gov/hmrc/agentoverseasapplicationfrontend/controllers/ApplicationControllerISpec.scala [hunk 3]
```diff
@@ -423,6 +427,74 @@ class ApplicationControllerISpec extends BaseISpec {
...
+      val authenticatedRequest = cleanCredsAgent(FakeRequest())
+      val result = await(controller.showUkTaxRegistrationForm(authenticatedRequest))
+
+      status(result) shouldBe 303
+      result.header.headers(LOCATION) shouldBe routes.ApplicationController.showAntiMoneyLaunderingForm().url
+    }
+  }
+
+  "POST /personal-details" should {
+    "store choice in session after successful submission and redirect to next page" in {
```
Comment (arkowalski) [inline it/uk/gov/hmrc/agentoverseasapplicationfrontend/controllers/ApplicationControllerISpec.scala]: add tests for the basic standard errors, no radio selected, field value is empty

Comment (rvijayan) [reply it/uk/gov/hmrc/agentoverseasapplicationfrontend/controllers/ApplicationControllerISpec.scala]: validations are not part of this ticket. Will be dealt in ASA-301.
